gpio_local.py

```diff
@@ -0,0 +1,29 @@
+from pin import Pin, MAPPINGS
```

Comment:
For executable `.py` files, make sure to:
* Add a shebang for python, to be easily executable
* Put your entrypoint code in a `__main__`

Reply:
Updating